core/utils/style.ts

```diff
@@ -109,17 +106,11 @@ function getStyle(element: Element, style: string): string {
  * @alias Blockly.utils.style.getComputedStyle
  */
 export function getComputedStyle(element: Element, property: string): string {
```

Comment:
Why is this exported but `getStyle` is not? Do we want to deprecate this and then stop exporting it in v10?

Reply:
I can't delete it because we use it in `flyout_button.ts`, but I can deprecate it and leave myself a note to make it `@internal` instead of deleting it in the next release.

Reply:
Actually, I don't like that because it means we have deprecation warnings in the browser for normal things (flyouts with buttons) for a full release.

Reply:
Can we export `getStyle` instead, and use that? (since it seems like the more general-purpose function anyway)

Reply:
But then we're just dropping one export to add another, which I don't like.

I'm looking into the uses of `getStyle` to see when one or the other gets used and whether I can simplify more.

Reply:
We ended up deleting `getStyle` since `element.getComputedStyle` returns a superset of the values on `element.style`, so the function would always short circuit before reaching the `||`. I.e. it wasn't doing anything different from `getComputedStyle`.